<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image:url(:/newPrefix/fundal aplicatie.jpg);
}</string>
   </property>
   <widget class="QTableWidget" name="tabelvalori">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>981</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>75</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>φ</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>λ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>O</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Da</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dg</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Δg</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Δt</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>f</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabelrezultate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>981</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>75</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>φ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>λ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>O</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>m</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ml</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Da</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dg</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ẟc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Δc</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="titlumare">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>721</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Calculator de navigație estimată&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="adaugasecv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:400;&quot;&gt;Adaugă o secvență&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlutabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Tabel date inițiale&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlutabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Tabel rezultate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculeaza">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>840</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;Bookman Old Style&quot;;
text-decoration: underline;
background-color:rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>CALCULEAZĂ</string>
    </property>
   </widget>
   <widget class="QLabel" name="sterge">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>85</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Șterge o secvență&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>85</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#plus{
background-color:rgb(170, 255, 255);
font: 25 15pt &quot;Bookman Old Style&quot;;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>85</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#minus{
background-color:rgb(170, 255, 255);
font: 25 15pt &quot;Bookman Old Style&quot;;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="fundal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#fundal{
border-image:url(:/newPrefix/fundal aplicatie.jpg);

}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="d2022text">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>90</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;d&lt;/span&gt;&lt;span style=&quot; font-size:26pt; vertical-align:sub;&quot;&gt;2022 = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="d2022">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>100</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 26pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>240</y>
      <width>341</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>160</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Simularea drumului navei pe harta de navigație &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;de la Nos Kaliakra la Delta Dunării&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homebutton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#homebutton{
background-color:rgb(0, 85, 255);
font: 25 9pt &quot;Bookman Old Style&quot;;
color:white
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exemplubutton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#exemplubutton{
background-color:rgb(0, 85, 255);
font: 25 9pt &quot;Bookman Old Style&quot;;
color:white
}</string>
    </property>
    <property name="text">
     <string>Exemplu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajutorbutton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#ajutorbutton{
background-color:rgb(0, 85, 255);
font: 25 9pt &quot;Bookman Old Style&quot;;
color:white
}</string>
    </property>
    <property name="text">
     <string>Ajutor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desprebutton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#desprebutton{
background-color:rgb(0, 85, 255);
font: 25 9pt &quot;Bookman Old Style&quot;;
color:white
}</string>
    </property>
    <property name="text">
     <string>Despre</string>
    </property>
   </widget>
   <zorder>fundal</zorder>
   <zorder>tabelvalori</zorder>
   <zorder>tabelrezultate</zorder>
   <zorder>titlumare</zorder>
   <zorder>adaugasecv</zorder>
   <zorder>titlutabel1</zorder>
   <zorder>titlutabel2</zorder>
   <zorder>calculeaza</zorder>
   <zorder>sterge</zorder>
   <zorder>plus</zorder>
   <zorder>minus</zorder>
   <zorder>d2022text</zorder>
   <zorder>d2022</zorder>
   <zorder>graphWidget</zorder>
   <zorder>label</zorder>
   <zorder>homebutton</zorder>
   <zorder>exemplubutton</zorder>
   <zorder>ajutorbutton</zorder>
   <zorder>desprebutton</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../licenta/venv/fundaltransp.qrc"/>
 </resources>
 <connections/>
</ui>
